test/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "tsBuildInfoFile": "../build/tsbuildinfo/test",
     "types": ["node", "mocha", "flot"]
   },
-  "include": ["**/*", "../typings/elastic__node_crypto.d.ts", "typings/**/*"],
+  "include": ["**/*", "../typings/elastic__node_crypto.d.ts", "typings/**/*", "../packages/kbn-test/types/ftr_globals/**/*"],
```

Comment:
Went ahead and unified the global mocha and snapshot types under this directory in `@kbn/test` and included them in both the test/tsconfig.json and x-pack/test/tsconfig.json projects.